<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorForm</class>
 <widget class="QWidget" name="EditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayoutMain">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelBand">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Band:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="labelDateFrom">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Date from:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinBoxVersion">
       <property name="minimumSize">
        <size>
         <width>71</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>71</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximum">
        <number>63</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelVersion">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxBand">
       <property name="minimumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>51</width>
         <height>21</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QDateTimeEdit" name="dateTimeEditEnd">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="labelDateTo">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Date to:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QDateTimeEdit" name="dateTimeEditBegin">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAcept">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
